lib/improver/utilities/cube_manipulation_new.py

```diff
@@ -179,34 +291,12 @@ def concatenate_cubes(
 
     Returns:
         result (Iris cube):
-            Concatenated / merge cube.
-
+            Concatenated cube.
     """
-    if coords_to_slice_over is None:
-        coords_to_slice_over = ["realization", "time"]
-    if coordinates_for_association is None:
-        coordinates_for_association = ["forecast_reference_time",
-                                       "forecast_period"]
-    if isinstance(cubes_in, iris.cube.Cube):
-        cubes = iris.cube.CubeList([cubes_in.copy()])
-    else:
-        cubes = iris.cube.CubeList([])
-        for cube in cubes_in:
-            cubes.append(cube.copy())
-
-    for coord_to_slice_over in coords_to_slice_over:
-        cubes = _slice_over_coordinate(cubes, coord_to_slice_over)
-
-    cubes = _equalise_cubes(cubes, merging=False)
-
-    associated_master_cubelist = iris.cube.CubeList([])
-    for cube in cubes:
-        associated_master_cubelist.append(
-            _associate_any_coordinate_with_master_coordinate(
-                cube, master_coord=master_coord,
-                coordinates=coordinates_for_association))
-
-    result = associated_master_cubelist.concatenate_cube()
+    plugin = ConcatenateCubes(
```

Comment:
This is something I'm undecided about - I was hoping reviewers might have some thoughts.  The reason to keep it for the moment was mostly to show that the functionality was not changed by the refactor.  Keeping it for a little while would reduce changes to the code base, but otherwise wouldn't have much intrinsic value.

Reply:
I think that it's fine to keep this function for the moment, and we can tidy things up, by calling the ConcatenateCubes class directly when the bulk of this work is complete.